ckan/lib/task.py

```diff
@@ -0,0 +1,101 @@
+from rq import Queue
+from redis import Redis
+
+from logging import getLogger
+log = getLogger(__name__)
+
+redis_conn = Redis()
```

Comment:
We should late-bind this (assuming lib doesn't already late-bind) and move it into init_queues()